src/renderRepoCard.js

```diff
@@ -49,6 +50,14 @@ const renderRepoCard = (repo, options = {}) => {
   const totalStars = kFormatter(stargazers.totalCount);
   const totalForks = kFormatter(forkCount);
 
+  const templateBadge = `
+    <g data-testid="template-badge" transform="translate(320, 24)">
```

Comment:
transform y should be 38 i think same as the archiveTemplate?

Reply:
if y=38 it wont' be proper aligned. Do you have archive use case?